build/markdown.m4

```diff
@@ -49,8 +49,8 @@ AC_DEFUN([GP_CHECK_MARKDOWN],
     GTK_VERSION=2.16
     WEBKIT_VERSION=1.1.13
 
-    GP_CHECK_GTK3([webkit_package=webkitgtk-3.0],
-                  [webkit_package=webkit-1.0])
+    GP_CHECK_GTK3([webkit_package=webkit2gtk-4.0],
+                    [webkit_package=webkit2gtk-4.0])
```

Comment:
This won't work for a GTK2 Geany: webkit2gtk is just not available for GTK2

Reply:
Might be time to abandon GTK2 Geany. :trollface:

Reply:
yes webkit2gtk , only have support for GTK3 , in Fedora Linux is removing webkit1gtk2 from repos (manly because security issues)